src/platform/with_egl/native_gl_context.rs

```diff
@@ -2,9 +2,20 @@ use euclid::Size2D;
 use platform::NativeGLContextMethods;
 use platform::with_egl::utils::{create_pixel_buffer_backed_offscreen_context};
 use std::ffi::CString;
+use std::ops::Deref;
 use egl;
 use egl::types::{EGLint, EGLBoolean, EGLDisplay, EGLSurface, EGLConfig, EGLContext};
+use libloading as lib;
 
+lazy_static! {
+    static ref GL_LIB: Option<lib::Library>  = {
+        if cfg!(target_os = "android") {
+            lib::Library::new("libGLESv2.so").ok()
```

Comment:
That would require a change in the API because get_proc_address trait is static and we can't know the user preference. We could work on that on a separate issue/PR, it's less critical in this moment.